requirements.txt

```diff
@@ -30,7 +30,7 @@ open-clip-torch==2.22.0
 git+https://github.com/Nerogar/mgds.git@848436a#egg=mgds
 
 # xformers
-xformers==0.0.22.post4
+xformers==0.0.22.post4+cu118
```

Comment:
I've made it explicit here that we want xformers for CUDA 11.8 so we don't run into #55 again.